weave/trace_server/trace_server_interface.py

```diff
@@ -191,6 +191,10 @@ class CallsQueryReq(BaseModel):
     sort_by: typing.Optional[typing.List[_SortBy]] = None
     query: typing.Optional[Query] = None
 
+    # TODO: type this with call schema columns
+    # optional column_specification, selection and order
+    columns: typing.Optional[typing.List[str]] = None
```

Comment:
this string follows the same rules as `_SortBy.field` and `GetFieldOperator.get_field_`. No action is required here, but it would be nice to somehow centralize the comments and spec - maybe just add a comment note here.

```

```